api/app/src/main/resources/db/migration/V20241206__add_latest_display_name_column_to_packet_group.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE "packet_group"
+    ADD COLUMN "latest_display_name" text DEFAULT '';
+
+ALTER TABLE "packet_group"
+    ALTER COLUMN "latest_display_name" SET NOT NULL;
+
+ALTER TABLE "packet_group"
+    ALTER COLUMN "latest_display_name" DROP DEFAULT;
```

Comment:
We always want the `latest_display_name`columns to be populated with non-empty values  don't we? So perhaps here we should also update any existing packet groups with display name. However if we were going to do that properly (not just copying over latest packet name) we'd need to get into the metadata, which would need to be done in code (getting metadata from outpack server). One for another ticket?